src/frontier_exploration/scripts/frontier_filter.py

```diff
@@ -90,6 +95,49 @@ def init_markers(self):
 
         return points_clust
 
+    def is_valid_frontier(self, node):
+        if node[0] > -0.5 and node[0] < 12.0 and node[1] > -10.0 and node[1] < 2.0:
```

Comment:
What are these magic numbers? I am guessing this is a geofence? Please convert to macros or variables and add comments

Reply:
geofence